core/src/main/java/org/mini2Dx/core/assets/AssetManager.java

```diff
@@ -196,7 +210,35 @@ public float getProgress() {
 		return completedTasks / queuedAssets;
 	}
 
+	/**
+	 * Returns the number of all assets (both loaded and not loaded) ever queued
+	 * @return A value more than or equal zero
+	 */
+	public float getQueuedAssets() {
+		return queuedAssets;
+	}
+
+	/**
+	 * Returns the number of already loaded assets
+	 * @return A value more than or equal zero
+	 */
+	public float getCompletedTasks() {
+		return completedTasks;
+	}
+
+	/**
+	 * Returns the number of currently queued assets
+	 * @return A value more than or equal zero
+	 */
+	public float getUnfinishedTasks() {
```

Comment:
why do we need floats here?
there shouldn't be e.g. 1.5 tasks unfinished

Reply:
There could be 1.5 tasks finished as a task might be 50% complete if it runs over several frames or asynchronously.

Reply:
I see what you mean, but currently there are only whole-number increments of this number